src/assets/styles/generic/_vuetify.scss

```
@@ -40,12 +40,12 @@ html {
 }
 
 .v-theme--light {
-  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat),  {
+  .v-btn--disabled:not(.v-btn--icon):not(.v-btn--flat).v-btn--variant-elevated  {
```

The button cannot be `flat` and `elevated` at the same time, so I think you can simplify the selector. Same for other selectors below.

```suggestion
  .v-btn--disabled:not(.v-btn--icon).v-btn--variant-elevated  {
```